app/src/main/res/layout/account_select_linear.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:paddingStart="?android:attr/listPreferredItemPaddingStart"
+        android:paddingEnd="?android:attr/listPreferredItemPaddingEnd"
+        android:textStyle="bold"
+        android:text="@string/switch_account"
+        android:textSize="20sp"
+        android:textColor="?attr/textColor"
+        android:layout_width="match_parent"
+        android:layout_rowWeight="1"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/account_recycler_view"
+        android:orientation="horizontal"
+        android:descendantFocusability="afterDescendants"
+        android:layout_marginTop="10dp"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+        tools:itemCount="4"
+        tools:listitem="@layout/account_list_item" />
+
+    <com.google.android.material.button.MaterialButton
```

Comment:
The style you are using is only used for a specific kind of dialog, this is not such a dialog and such should instead of the standard blackbutton/whitebutton. Also this for some reason has a layout_marginStart=10dp that causes misaligned + it does not take up the entire bottom making the hitbox a bit fucked impo
```xml
    <com.google.android.material.button.MaterialButton
        android:id="@+id/manage_accounts_button"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_gravity="center"
        android:layout_marginBottom="10dp"
        android:text="@string/manage_accounts"
        android:textSize="16sp"
        app:icon="@drawable/ic_baseline_edit_24"
        style="@style/BlackButton" />
```

Reply:
My concern with this is it looks fine on Phone, but with a background and the button being full length, it doesn't look good on TV at all in my opinion. I'll make the change though.

Reply:
@Luna712 If you feel like it does not look good on tv you can make a separate UI for that, however phone is still the target device for this project

Reply:
I understand. Thanks!